components/bt/host/bluedroid/btc/profile/std/sdp/btc_sdp.c

```diff
@@ -1132,7 +1132,7 @@ void btc_sdp_cb_handler(btc_msg_t *msg)
         break;
     case BTA_SDP_CREATE_RECORD_USER_EVT:
         param.create_record.status = p_data->status;
```

Comment:
Thanks for the review! I totally agree, I was just mimicking the behaviour on the `BTA_SDP_SEARCH_COMP_EVT`, were `p_data->status` is used instead of `p_data->sdp_search_comp.status`. There should be no difference in terms of accessed memory, but I will update both to make the code clearer.